CSharpOOP4/CSharpOOP4/Ball.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP4
+{
+    public class Ball : Shape
+    {
+        public double Radius
+        {
+            get => _radius;
+            set { _radius = value; }
+        }
+        private double _radius;
+        public double ballVolume;
+
+        public Ball(string name, double radius) : base(name)
+        {
+            Name = "Ball";
```

Comment:
You should use name, that you pass to the constructor when you create the object. Fix for all classes.